zulipterminal/model.py

```diff
@@ -567,8 +567,14 @@ def update_stream_message(
         if response["result"] == "success":
             old_topic = self.index["messages"][message_id].get("subject", None)
             new_topic = request["topic"]
+            stream_name = self.index["messages"][message_id].get(
+                "display_recipient", None
+            )
+            no_of_messages = len(self.index["edited_messages"])
```

Comment:
This is all the edited messages we know about, so this may work the first time, with no known edited messages, but I expect likely will grow each time and so report the wrong result.

Try testing this manually, and then you can add a test case for this and check it fails, before fixing.

Reply:
I got your point. Therefore, I have used different method -
1. Getting all stream messages
2. Filtering messages based on topic where message is edited by user.
3. Compare message_id of edited message and topic messages to get recent_edited_messages.